src/Commands/viewProperties.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Commands folder is new added right? Does it follow the existing code structure?

Reply:
There's no commands folder previously and the existing commands are distributed in various xxxExplorer. I tries to leverage the commands folder to make my code structure more clear.